### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by digeff**
Added localize call to all the strings that might be shown to the user

(Most errors can produce a failed request, which will be sent as the message of a DebugProtocol.response which can be shown to the user)

### CODE REVIEW

File: src/chrome/chromeDebugAdapter.ts [hunk 1]
```diff
@@ -643,7 +643,7 @@ export class ChromeDebugLogic {
     }
 
     protected threadName(): string {
-        return 'Thread ' + ChromeDebugLogic.THREAD_ID;
+        return localize('thread.prefix', 'Thread ') + ChromeDebugLogic.THREAD_ID;
```
Comment (roblourens) [inline src/chrome/chromeDebugAdapter.ts]: Space should go outside of the localized string so it isn't lost in localization

Comment (roblourens) [reply src/chrome/chromeDebugAdapter.ts]: Actually the id should be an arugument like {0} in case it goes before "Thread" in some language

Comment (digeff) [reply src/chrome/chromeDebugAdapter.ts]: Changed to return localize('thread.prefix', 'Thread {0}', ChromeDebugLogic.THREAD_ID);

File: src/chrome/internal/breakpoints/features/bpsDeltaCalculator.ts [hunk 2]
```diff
@@ -89,8 +89,8 @@ export class BPRsDeltaCalculator {
                 existingToLeaveAsIs: this.printLocations(match.existingToLeaveAsIs)
             };
 
-            const additionalDetails = localize('\nRequested breakpoints = {0}\nExisting breakpoints = ${1}\nMatch = ${2}', JSON.stringify(this._requestedBPRecipes.breakpoints.map(canonicalizeBPLocation)), JSON.stringify(Array.from(this._currentBPRecipes).map(canonicalizeBPLocation)), JSON.stringify(matchJson));
-            throw new Error(localize('error.deltaCalculator.deltaCalculatorInvalidResult', '{0}\nmatch: {1}', errorMessage, additionalDetails));
+            const additionalDetails = localize('error.deltaCalculator.invalidResultDetails.', '\nRequested breakpoints = {0}\nExisting breakpoints = ${1}\nMatch = ${2}', JSON.stringify(this._requestedBPRecipes.breakpoints.map(canonicalizeBPLocation)), JSON.stringify(Array.from(this._currentBPRecipes).map(canonicalizeBPLocation)), JSON.stringify(matchJson));
```
Comment (roblourens) [inline src/chrome/internal/breakpoints/features/bpsDeltaCalculator.ts]: `.` in string id

Comment (digeff) [reply src/chrome/internal/breakpoints/features/bpsDeltaCalculator.ts]: Fixed

**Comment by digeff** [comment]
Will re-open when it's ready to be merged

**Pull request closed without merging**